### PR TITLE
boxfort: add python 3.{8,9} support

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks

### CODE REVIEW

**Comment by rafaelmartins** [comment]
cram patch looks ok

File: dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+DEPEND="test? (
+		$(python_gen_any_dep 'dev-util/cram[${PYTHON_USEDEP}]')
+	)"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/BoxFort-${BOXFORT_COMMIT}"
+
+python_check_deps() {
+	has_version "dev-util/cram[${PYTHON_USEDEP}]"
```
Comment (thesamesam) [inline dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild:30]: use test &&?

File: dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit meson python-any-r1
+
+BOXFORT_COMMIT="ac0507b3f45fe58100b528baeb8ca04270b4a8ff"
+
+DESCRIPTION="Convenient & cross-platform sandboxing C library"
+HOMEPAGE="https://github.com/Snaipe/BoxFort"
+SRC_URI="https://github.com/Snaipe/BoxFort/archive/${BOXFORT_COMMIT}.tar.gz -> boxfort-${BOXFORT_COMMIT}.tar.gz"
```
Comment (thesamesam) [inline dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild:14]: You know I'm going to say I prefer -> ${P}.tar.gz ;)

File: dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild (hunk 1 above)
Comment (hydrapolic) [reply dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild]: Good catch, haven't noticed :)

File: dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild (hunk 2 above)
Comment (hydrapolic) [reply dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild]: Ok, changed :)

File: dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	has_version "dev-util/cram[${PYTHON_USEDEP}]"
+}
+
+pkg_setup() {
+	use test && python-any-r1_pkg_setup
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dsamples=$(usex test true false)
```
[the remark this answers is not in the document]
Comment (hydrapolic) [reply dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild:39]: Yes, it only accepts true/false.

**Comment by hydrapolic** [comment]
Thanks for the review @thesamesam, pushed an update.

**Comment by thesamesam** [comment]
@hydrapolic No problem. Is it intentional that boxfort only installs a static library, not a dynamic one? (this is possible given it is for sandboxing, but I thought I should ask.)

**Comment by hydrapolic** [comment]
> @hydrapolic No problem. Is it intentional that boxfort only installs a static library, not a dynamic one? (this is possible given it is for sandboxing, but I thought I should ask.)

Yes, it's really so. Haven't noticed so far though.

**Comment by thesamesam** [comment]
@hydrapolic Fab, thank you as ever!

**Comment by hydrapolic** [comment]
Thank you @thesamesam

**Pull request closed without merging**